tests/gflownet/envs/common.py

```diff
@@ -359,6 +359,8 @@ def test__forward_actions_have_nonzero_backward_prob(env):
 
 def test__backward_actions_have_nonzero_forward_prob(env, n=1000):
     states = _get_terminating_states(env, n)
+    if states is None:
+        return
```

Comment:
I am surprised states can be None since `get_random_terminating_states()` is now implemented in the base env and hence is available to all states :confused:.

Reply:
I can't really comment on that. I just saw that it was None for the crystal env and that every other test had a mechanism for dealing with Nones except this one.
I guess this means that the true underlying issue isn't fixed and that other tests (which already had the "`if states is None`") might also have issues.

Reply:
Ok, thanks, I will take a closer look at this.

Reply:
I have kept the check-and-return-if-None but added warnings so that this doesn't go silent at least: https://github.com/alexhernandezgarcia/gflownet/pull/204/commits/993f4568fd8c466244d873de2b7133e9b21a6347